slides/shared/about-slides.md

```diff
@@ -6,12 +6,12 @@
 
 - You can get updated "builds" of the slides there:
 
-  http://container.training/
+  @@SLIDES@@
```

Comment:
We want to use container.training here, since we want to send people to the _updated_ slides (the big list of all slides).